Patch to fix compilation errors by removing the java examples target when building for java

Index: project/build/java.mk
===================================================================
--- project/build/java.mk	(revision 1551)
+++ project/build/java.mk	(working copy)
@@ -2,7 +2,11 @@
 # java of swig
 #
 
+ifeq (${SWIG_OSNAME}, linux)
+JAVA_BUILD              = ${JAVA_DLL} ${JAVA_OBJ_DIR}/${JAVA_JAR}
+else
 JAVA_BUILD		= ${JAVA_DLL} ${JAVA_OBJ_DIR}/${JAVA_JAR} ${JAVA_EXAMPLES_CLASSES}
+endif
 
 # Include/combine the follwoing source into the actual JAVA_BRIDGE
 # line to enable the streams prototype for Java
